Do not try to create the same function multiple time

We declare multiple triggers which execute the same function.

The bundle tries to generate the function as many times as she is used in triggers, so MakerBundle is triggering this error:

> [ERROR] The file "path/to/migrations/triggers/functions/tg_function_name.sql" can't be generated because it already exists.

--- a/src/Factory/TriggerCreator.php
+++ b/src/Factory/TriggerCreator.php
@@ -137,11 +137,14 @@ final class TriggerCreator implements TriggerCreatorInterface
                 'definition' => $resolvedTrigger->definition,
             ];
 
-            $this->generator->generateFile(
-                sprintf('%s/functions/%s.sql', $storageDirectory, $resolvedTrigger->function),
-                __DIR__ . '/../Symfony/Maker/Resources/skeleton/sql/postgresql_function.tpl.php',
-                $params
-            );
+            $functionFilePath = sprintf('%s/functions/%s.sql', $storageDirectory, $resolvedTrigger->function);
+            if (!file_exists($functionFilePath)) {
+                $this->generator->generateFile(
+                    $functionFilePath,
+                    __DIR__ . '/../Symfony/Maker/Resources/skeleton/sql/postgresql_function.tpl.php',
+                    $params
+                );
+            }
 
             $this->generator->generateFile(
                 sprintf('%s/triggers/%s.sql', $storageDirectory, $resolvedTrigger->name),

